src/Bookings.js

```diff
@@ -1,18 +1,47 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+// import Form from "./Form.js";
 
 const Bookings = () => {
+  const [data, setData] = useState([]);
+  const [searchVal, setSearchVal] = useState("");
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  // let booking = () =>{
+  //   setBookings(bookings.concat(FakeBookings));
+  // }
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/delayed`).then(res => {
+      if (res.ok) {
```

Comment:
Hhandling the errors, nice! However theres another possibility here, do you know what it is?

The fetch function can actually throw an error too in some cases, what should we do about it? 😄